tests/vxlan/test_vxlan.py

```diff
@@ -0,0 +1,653 @@
+#! /usr/bin/env python3
```

Comment:
Can we rename this to` test_vxlan_ecmp.py`?` test_vxlan `is generic which may not be the right one for this use-case

Reply:
Will do.